assets/js/experience.js

```diff
@@ -1,3 +1,62 @@
+//  Work experience cards
+
+
+const experiencecards = document.querySelector(".experience-cards");
+const exp = [
+  {
+    title: "Reasearch and Development Intern",
+    cardImage: "assets/images/experience-page/UCL.JPG",
+    place: "UCL",
+    time: "(May2020-present)",
+    desp: "<li>Automatic detection of Orthopedic Implants using Deep Learning.</li> <li>Collection of radiographs, data cleaning and database management.</li> <li>Image Quality Analysis for Bio-Medical Images.</li> <li>Serving Models using TensorFlow JS.</li> <li>Experimenting with Few Shot Learning, Adversarial Training and Class Activation Maps.</li>",
+  },
+  {
+    title: "Robotics and Embedded system Intern",
+    cardImage: "assets/images/experience-page/lema-labs.png",
+    place: "LEMA LABS",
+    time: "(Jun-Jul,2019)",
+    desp: "<li>Design low cost prototyping boards for teaching robotics to underprivileged children.</li> <li>Worked on integration of Face Recognition based models with Raspberry Pi and Intel Movidius.</li> <li>Image Quality Analysis for Bio-Medical Images.Experimented with various algorithms for Lane Detection as well as Shortest Paths for Autonomous Vehicles.</li>",
+  },
+  {
+    title: "Research Intern",
+    cardImage: "assets/images/experience-page/ISI.png",
+    place: "ISI, Kolkata",
```

Comment:
As I suggested earlier, please replace all three of this with some other data. You can use Deedy Resume for reference. The project should serve as a generalized template for everyone. Not as my own portfolio being open sourced

Reply:
Can I do this writing part in the next PR as this PR includes only the dynamic part?

Reply:
okay